backend/src/utils/types.ts

```diff
@@ -0,0 +1,40 @@
+type User = {
+    id : string,
+    role : string,
+    email : string
+}
+
+type Dish = {
+    id : string,
+    qid : number,
+    registered : Date,
+    type: string
+}
+
+// need more info on returned type
+type Transaction = {
+    id : string,
+    dishID : string,
+    userID: string,
+    returned : any,
+    timestamp : Date
+}
+
+enum DishStatus {
+    overdue =  "Overdue",
+    inUse = "In Use",
+    returned = "Returned",
+    broken = "Broken",
+    lost = "Lost"
+}
+
+type DishTableVM = {
+    id : string,
+    type : string,
+    status : DishStatus,
+    overdue : number,
+    timesBorrowed : number,
+    dateAdded : Date
+}
+
+export {User, Dish, Transaction, DishStatus, DishTableVM}
```

Comment:
nit:
instead of exporting all the types explicitly at the bottom add export keyword in front of the types that you want to export